lib/AcqUnits/AcqUnitsField/AcqUnitsField.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import user from '@testing-library/user-event';
+
+import stripesFinalForm from '@folio/stripes/final-form';
+
+import AcqUnitsField from './AcqUnitsField';
+
+const UNITS = [{ id: '1', name: 'unit 1' }, { id: '2', name: 'unit 2' }];
+
+// eslint-disable-next-line react/prop-types
+const renderForm = () => (
+  <form>
+    <AcqUnitsField
+      id="acq-units"
+      isFinal
+      name="acq-units"
+      units={UNITS}
+    />
+  </form>
+);
+
+const FormCmpt = stripesFinalForm({})(renderForm);
+
+const renderComponent = (props = {}) => (render(
+  <MemoryRouter>
+    <FormCmpt onSubmit={() => { }} initialValues={{ 'acq-units': [] }} {...props} />
+  </MemoryRouter>,
+));
+
+describe('AcqUnitsField', () => {
+  it('should assign acq unit', () => {
+    renderComponent();
+    expect(screen.getByText('0 items selected')).toBeDefined();
```

Comment:
don't think you need expects before click

Reply:
why? we check for default result and check result after

Reply:
it's not mapped directly to the AAA pattern, but might be useful in case default behavior is changed, IMHO

Reply:
but you check after expected result, I don't reason to check initial state